stable_baselines/acktr/acktr_disc.py

```diff
@@ -100,6 +102,11 @@ def __init__(self, policy, env, gamma=0.99, nprocs=1, n_steps=20, ent_coef=0.01,
         if _init_setup_model:
             self.setup_model()
 
+    def _get_pretrain_placeholders(self):
+        policy = self.train_model
+        if isinstance(self.action_space, Discrete):
+            return policy.obs_ph, self.action_ph, policy.policy
+
```

Comment:
This should not be able to return None.

I assume this is what you ment?
```suggestion
        raise NotImplementedError('Only discrete actions are supported for ACKTR for now')

```

Reply:
The thing is that this exception will be raised (normally) before the call to `_get_pretrain_placeholders()`, so I don't know if I should raise the same exception a second time ?

Reply:
then why is it checked?

Reply:
true, I will add the exception